doc/manual/oqum/risk/02_config_event_based_risk.tex

```diff
@@ -124,3 +124,69 @@
   \caption{Example combined configuration file for running a stochastic event based risk calculation starting from a precomputed set of ground motion fields (\href{https://raw.githubusercontent.com/gem/oq-engine/master/doc/manual/oqum/risk/verbatim/config_gmf_event_based_risk.ini}{Download example})}
   \label{lst:config_gmf_event_based_risk}
 \end{listing}
+
+
+\paragraph{Additional parameters}
+
+A few additional parameters related to the event based risk calculator that
+may be useful for controlling specific aspects of the calculation are listed
+below:
+
+\begin{itemize}
+
+  \item \Verb+individual_curves+: this boolean parameter is used to specify if
+    the asset loss curves for each branch realization should be saved to the 
+    datastore. For the asset loss curves output, by default the engine only 
+    saves and exports statistical results, i.e. the mean and quantile asset 
+    loss curves. If you want the asset loss curves for each of the individual 
+    branch realizations, you must set \Verb+individual_curves=true+ in the job 
+    file. Please take care: if you have hundreds of realizations, the data 
+    transfer and disk space requirements will be orders of magnitude larger 
+    than just returning the mean and quantile asset loss curves, and the 
+    calculation might fail. The default value of \Verb+individual_curves+ is 
+    \Verb+false+.
+
+  \item \Verb+asset_correlation+: if the uncertainty in the loss ratios
+    has been defined within the \gls{vulnerabilitymodel}, users can specify
+    a coefficient of correlation that will be used in the Monte Carlo sampling
+    process of the loss ratios, between the assets that share the same
+    \gls{taxonomy}. If the \Verb+asset_correlation+ is set to one,
+    the loss ratio residuals will be perfectly correlated. On the other hand,
+    if this parameter is set to zero, the loss ratios will be sampled
+    independently. If this parameter is not defined, the
+    \glsdesc{acr:oqe} will assume zero correlation in the vulnerability. As of
+    \glsdesc{acr:oqe18}, \Verb+asset_correlation+ applies only to continuous
+    \glspl{vulnerabilityfunction} using the lognormal or Beta distribution; 
+    it does not apply to \glspl{vulnerabilityfunction} defined using the PMF
+    distribution. Although partial correlation was supported in previous
+    versions of the engine, beginning from \glsdesc{acr:oqe22}, values between
+    zero and one are no longer supported due to performance considerations. The
+    only two values permitted are \Verb+asset_correlation = 0+ and 
+    \Verb+asset_correlation = 1+.
+
+  \item \Verb+asset_loss_table+: this parameter
+    specifies whether the individual asset losses should be stored for every 
+    event for each realization; the default value of this parameter is
+    \Verb+false+. If this flag is set to \Verb+true+, a
+    matrix containing all of the losses is saved in the datastore.
+
```

Comment:
I am not sure if I want to restore this parameter. For sure, I do not want it documented.

Reply:
Removed asset_loss_table from both scenario_risk and event_based_risk sections